factories/cases.ts

```diff
@@ -48,3 +48,10 @@ export const mockedCaseNote = caseFactory.build({
     case_note_title: 'i am a case title',
   },
 });
+export const mockedWarningNote = caseFactory.build({
```

Comment:
We already have a mocked warning note in the codebase. Check `./factories/warningNotes`. perhaps we could reuse that?

Reply:
I tried to see if we could reuse this but couldn't get it to work quickly. 😅 

From what I understand, what we want here is slightly different to an actual warning note. Since when we render the records history, we're getting the Warning Note _CaseNote_, which would not have the actual warning note details within it but have form data.

I could rename this to be a bit more descriptive. 🤔

Reply:
Yeah – I think you could import the factory from `./factories/warningNotes` and create your own using `.build` that way. If I'm honest the way our fixtures/factories works is a little inconsistent anyway, so if this is worthing for you I wouldn't worry too much!